pkg/cmd/pulumi/policy_new.go

```diff
@@ -297,7 +297,7 @@ func choosePolicyPackTemplate(templates []workspace.PolicyPackTemplate,
 	if err := survey.AskOne(&survey.Select{
 		Message:  message,
 		Options:  options,
-		PageSize: len(options),
+		PageSize: 10,
```

Comment:
we should use the height of the user's terminal to determine the size of the page

it looks good inside vscode, but leaves a bit to be desired in a full sized terminal.
![2022-07-15-165446_2560x1051_scrot](https://user-images.githubusercontent.com/25750473/179325537-ab836e17-b768-4e4a-9015-3dcc720e9856.png)
.

Reply:
Just tested it, definitely looks better but there's the issue with the `Please choose a template (#/#) shown:` prompt appearing numerous times as you scroll again

Reply:
Actually, it looks like providing a small buffer (terminal height - 5) solves that, but the flickering is 😠 
<img width="1404" alt="Screen Shot 2022-07-15 at 5 07 24 PM" src="https://user-images.githubusercontent.com/54565933/179326171-d159260b-0abb-43d1-9503-62c464673a28.png">

Reply:
I would be inclined to set a lower limit to reduce flickering. Is there an upstream issue to track that flickering?

Reply:
I meant with the CLI library - is this something other folks have noticed or the upstream repo is aware of? The flickering is very noticeable - even more-so on Windows - and I think it'd be good to flag that with the maintainer.

Reply:
Ah no, it's also still there even after upgrading to their v2; I'll open an issue now.